nix/shell.sh

```diff
@@ -51,10 +51,12 @@ function integ {
         echo "- renegotiate_apache"
         echo "   This test requires apache to be running. See codebuild/bin/s2n_apache.sh"
         echo "    for more info."
+        echo "- sslyze (on aarch64)"
+        echo "   Test tooling isn't available for arm"
         return
     fi
     if [[ -z "$1" ]]; then
-        banner "Running all integ tests except cross_compatibility, renegotiate_apache."
+        banner "Running all integ tests except cross_compatibility, renegotiate_apache, and sslyze on arm."
         (cd $SRC_ROOT/build; ctest -L integrationv2 -E "(integrationv2_cross_compatibility|integrationv2_renegotiate_apache)" --verbose)
```

Comment:
Should you mention sslyze here? Calling out the other two make sense, since Nix specifically skips them, but sslyze is skipped by cmake.

Reply:
that's fair, since we are doing a cmake warning.